x-pack/test/security_solution_cypress/cypress/e2e/investigations/alerts/ransomware_detection.cy.ts

```diff
@@ -24,33 +25,30 @@ describe('Ransomware Detection Alerts', { tags: ['@ess', '@serverless'] }, () =>
     });
   });
 
-  describe('Ransomware display in Alerts Section', () => {
+  after(() => {
+    cy.task('esArchiverUnload', 'ransomware_detection');
+  });
+
+  describe('Ransomware display on Alerts Page', () => {
     beforeEach(() => {
       login();
       visitWithTimeRange(ALERTS_URL);
       waitForAlertsToPopulate();
     });
 
-    describe('Alerts table', () => {
-      it('shows Ransomware Alerts', () => {
-        cy.get(ALERT_RULE_NAME).should('have.text', 'Ransomware Detection Alert');
-      });
+    it('should show Ransomware Alerts in alerts table', () => {
+      cy.get(ALERT_RULE_NAME).should('have.text', 'Ransomware Detection Alert');
     });
 
-    describe('Trend Chart', () => {
-      beforeEach(() => {
-        selectAlertsHistogram();
-      });
-
-      it('shows Ransomware Detection Alert in the trend chart', () => {
-        cy.get(ALERTS_HISTOGRAM_SERIES).should('have.text', 'Ransomware Detection Alert');
-      });
+    it('should show Ransomware Prevention Alert in the trend chart', () => {
+      selectAlertsHistogram();
```

Comment:
if you look at the implementation of this function ([here](https://github.com/elastic/kibana/blob/main/x-pack/test/security_solution_cypress/cypress/tasks/alerts.ts#L275)) it has a `{ force: true }` that we should remove